### PR TITLE
2742 - Document the more granular permissions for public links

### PR DESCRIPTION
**Pull request by settermjd**
Relates to https://github.com/owncloud/enterprise/issues/2742

### CODE REVIEW

File: modules/user_manual/pages/files/public_link_shares.adoc [hunk 1]
```diff
@@ -1,49 +1,47 @@
...
-| Show file listing
-| The option allows recipients to list the files in the shared folder. If you don’t give permission to view the files you have xref:files/webgui/sharing.adoc#creating-drop-folders[a drop folder].
+| Download / View / Edit
+| Allows recipients to view, download, edit, and delete the public link's contents.
+
+| Download / View / Upload
+| Allows recipients to view, download and upload the public link's contents.
+
+| Upload only (File Drop)
+| Allows recipients to create xref:files/webgui/sharing.adoc#creating-drop-folders[a drop folder], which can receive files from multiple recipients without revealing the contents of the folder.
```
Comment (phil-davis) [inline modules/user_manual/pages/files/public_link_shares.adoc]: Allows recipients to create - ?
maybe you mean:
Allows users to create...

Comment (settermjd) [reply modules/user_manual/pages/files/public_link_shares.adoc]: All that was done here was a small variation from the text in the modal dialog. That said, if "users" makes more sense, fine by me.

Comment (settermjd) [reply modules/user_manual/pages/files/public_link_shares.adoc]: Actually, would "users" or "share recipients" make more sense as well as being technically correct?

Comment (sharidas) [reply modules/user_manual/pages/files/public_link_shares.adoc]: Personally "share recipients" ( more explicit and less confusion ) looks ok to me. But if technically "users" is better than "share recipients", then I am fine with that too.

File: modules/user_manual/pages/files/public_link_shares.adoc [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
-files which already have been uploaded.
+As of ownCloud version 10.0.2 you can xref:files/webgui/sharing.adoc#creating-drop-folders[create Drop Folders], where users can upload files to a central location, but not be able to change any existing ones, nor see other files which already have been uploaded.
 
-[[creating-public-link-shares]]
 == Creating Public Link Shares
 
-To create a public link share, first view the Sharing Panel of the file
-or folder that you want to create a public link share for, and under
-*"Public Links"* click *"Create public link"*.
+To create a public link share, first view the Sharing Panel of the file or folder that you want to create a public link share for, and under *"Public Links"* click *"Create public link"*.
```
Comment (mmattel) [inline modules/user_manual/pages/files/public_link_shares.adoc]: Why not using `menu:` and `btn:` 
(needs `:experimental:`)

File: modules/user_manual/pages/files/public_link_shares.adoc [hunk 3]
```diff
@@ -1,49 +1,47 @@
...
-Then, as with other shares, provide the name in the *"Link Name"*
-field, and fill out the options that suit what you want the link to
-support. You can find details of what each option does below.
+Then, as with other shares, provide the name in the *"Link Name"* field, and fill out the options that suit what you want the link to support. 
+You can find details of what each option does below.
 
 image:public-link/public-link-settings.png[Create a public link - step two.]
 
-Finally, click *"Save"* to complete creation of the share. Now that the share is created, you can:
+Finally, click *"Save"* to complete creation of the share.
```
Comment (mmattel) [inline modules/user_manual/pages/files/public_link_shares.adoc]: Why not using `btn:`

File: modules/user_manual/pages/files/public_link_shares.adoc (hunk 1 above)
Comment (settermjd) [reply modules/user_manual/pages/files/public_link_shares.adoc]: Thanks, @sharidas. @phil-davis?

File: modules/user_manual/pages/files/public_link_shares.adoc (hunk 2 above)
Comment (settermjd) [reply modules/user_manual/pages/files/public_link_shares.adoc]: Fine by me.

File: modules/user_manual/pages/files/public_link_shares.adoc (hunk 1 above)
Comment (phil-davis) [reply modules/user_manual/pages/files/public_link_shares.adoc]: My point was that "share recipients" are not the ones who create a "drop folder". Ordinary users create a "drop folder". And then recipients (people who are given the link) can do the stuff in the 2nd half of the sentence.

Comment (settermjd) [reply modules/user_manual/pages/files/public_link_shares.adoc]: Ah, okay. Understood.

**Comment by settermjd** [comment]
@phil-davis, the sentence in question was updated, as per your feedback.

**Comment by settermjd** [comment]
@pmaier1 @JKawohl @sharidas can you review and leave feedback pls?

**Pull request merged**